src/sentry/issues/ownership/grammar.py

```diff
@@ -533,6 +533,7 @@ def add_owner_ids_to_schema(rules: list[dict[str, Any]], owners_id: dict[str, in
 def create_schema_from_issue_owners(
     project_id: int,
     issue_owners: str | None,
+    # TODO: remove this parameter in follow-up PR
     add_owner_ids: bool = False,
```

Comment:
left this here because the custom job in `getsentry` references it. so after this PR is deployed, i'll remove the reference in `getsentry`, then remove this entirely as a function arg